index.js

```diff
@@ -54,7 +77,19 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let ratingCount = 0;
+
+  if(movies.length === 0){
+  return ratingCount;
+}
+  
+  for (let i = 0; i < movies.length; i ++){
+    ratingCount += Number(movies[i].imdbRating);
+  }
+
+return (ratingCount/movies.length);
+}
```

Comment:
While this doesn't change our code we don't need the `()` here on 91.  The division operator will resolve and then the number will be returned.